test/cjs-import.test.cjs

```diff
@@ -1,6 +1,5 @@
 const assert = require('node:assert');
```

Comment:
If we're gonna do this - should probably use the baked in assertions as well?

Reply:
Sure, done.